charms/jupyter-ui/metadata.yaml

```diff
@@ -10,7 +10,8 @@ resources:
     type: oci-image
     description: 'Backing OCI image'
     auto-fetch: true
-    upstream-source: gcr.io/kubeflow-images-public/jupyter-web-app:vmaster-ge4456300
+    # upstream-source: gcr.io/kubeflow-images-public/jupyter-web-app:vmaster-ge4456300
```

Comment:
Should this be removed?

Reply:
Good catch. I'm actually going to see if I can get this merged in a reasonable time and just use the upstream image, instead of using rocks: https://github.com/kubeflow/kubeflow/pull/5686

Reply:
Actually, I updated this to have a TODO comment, along with using a custom rocks image, so we should be able to use it as-is.